lib/bgm.dart

```diff
@@ -13,11 +13,11 @@ import 'package:flutter/widgets.dart';
 /// to keep running.
 class Bgm extends WidgetsBindingObserver {
   bool _isRegistered = false;
-  AudioCache audioCache;
-  AudioPlayer audioPlayer;
+  late AudioCache audioCache;
+  AudioPlayer? audioPlayer;
```

Comment:
Where is `audioPlayer` set?

Reply:
On the play method, github is probably collapsing the section of the code where it is set.

Reply:
Ah, now I see, so the audiocache gives back a player